README.md

```diff
@@ -38,7 +38,11 @@
   </a>
 
   <a href="https://github.com/airtai/faststream/blob/main/LICENSE" target="_blank">
-    <img src="https://img.shields.io/github/license/airtai/faststream.png" alt="Github">
+    <img src="https://img.shields.io/github/license/airtai/faststream.png" alt="License">
+  </a>
+
+  <a href="https://github.com/airtai/faststream/blob/main/CODE_OF_CONDUCT.md" target="_blank">
```

Comment:
Not sure about another bage adding: there are too much already

Reply:
Shall I remove it then?
Please take a look at the newly added badge before answering though - https://github.com/airtai/faststream/tree/742-docs-add-code-of-conduct

Reply:
Loots good enough. We can remove it anytime if it will be required